pkg/operator/ceph/agent/agent.go

```diff
@@ -152,6 +160,37 @@ func (a *Agent) createAgentDaemonSet(namespace, agentImage, serviceAccount strin
 		},
 	}
 
+	// Add extra mounts if any given through environment
+	extraMounts := os.Getenv(extraMountsEnv)
+	if extraMounts != "" {
+		mounts := strings.Split(extraMounts, ",")
+		for _, mount := range mounts {
+			mountdef := strings.Split(mount, "=")
+			if len(mountdef) != 2 {
+				logger.Errorf("badly formatted extraMounts '%s'. The format should be 'mountname=/host/path:/container/path,mountname2=/host/path2:/container/path2'", extraMounts)
```

Comment:
I think it would be better to return an error here (instead of logging it here).

Reply:
Agreed. Will fix.